src/planscape/forsys/get_forsys_inputs.py

```diff
@@ -350,3 +431,65 @@ def _get_raster_pixel_geo(self, x: int, y: int) -> Polygon:
                        (xmin, ymin)))
         geo.srid = settings.CRS_FOR_RASTERS
         return geo
+
+    def _convert_clustered_merged_condition_rasters_to_input_df(
```

Comment:
can this share any code with _convert_merged_condition_rasters_to_input_df? 

this file is getting a bit long, any chance we can break it up?